src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/ArrayBuffer.cs

```diff
@@ -49,8 +49,10 @@ public void Dispose()
             }
         }
 
+        public int AvailableSpanLength => _bytes.Length - _availableStart;
```

Comment:
Nit: can we call this just AvailableLength?

Reply:
Ok